openlibrary/templates/recentchanges/merge-works/comment.html

```diff
@@ -0,0 +1,17 @@
+$def with (change, page=None)
```

Comment:
If we could DRY these templates, I think that would be nice, since they are very similar